src/main/webapp/resources/js/components/samples/SampleDetailViewer.tsx

```diff
@@ -40,6 +41,7 @@ function DisplaySampleDetails({
   projectId,
   displayActions,
   children,
+  refetch,
```

Comment:
Update JSDoc for this

Reply:
Updated in fbdd2f916028236b49258e8556483716ea6192b3